pxtrunner/renderer.ts

```diff
@@ -41,7 +41,7 @@ namespace pxt.runner {
     }
 
     function appendBlocks($parent: JQuery, $svg: JQuery) {
-        $parent.append($('<div class="ui content blocks"/>').append($svg));
+        $parent.append($('<div class="ui content blocks"><div class="right">BLOCKS</div></div>').append($svg));
```

Comment:
use lf("blocks") + css class to capitalize

Reply:
ok